lib/Jit/LLILCJit.cpp

```diff
@@ -313,8 +313,8 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
                << " method = " << Context.MethodName << '\n';
       }
       GcInfoAllocator GcInfoAllocator;
-      GCInfo GcInfo(&Context, MM.getStackMapSection(), MM.getHotCodeBlock(),
-                    &GcInfoAllocator);
+      GCInfo GcInfo(&Context, MM.getStackMapSection(), &GcInfoAllocator,
+                    *NativeEntry - MM.getHotCodeBlock());
```

Comment:
Should there be some kind of assert that we always get a positive number for the offset?